services/webhook/webhook.go

```diff
@@ -171,11 +171,29 @@ func PrepareWebhook(ctx context.Context, w *webhook_model.Webhook, event webhook
 
 	// Avoid sending "0 new commits" to non-integration relevant webhooks (e.g. slack, discord, etc.).
 	// Integration webhooks (e.g. drone) still receive the required data.
+	/** old code Commits
 	if pushEvent, ok := p.(*api.PushPayload); ok &&
 		w.Type != webhook_module.GITEA && w.Type != webhook_module.GOGS &&
 		len(pushEvent.Commits) == 0 {
 		return nil
 	}
+	*/
+
+	/**
+	 * new code Commits
+	 * @date 2023.06.04 23:55
+	 */
+	pushEvent, ok := p.(*api.PushPayload)
+	if ok {
+		// 字符串查找 有则返回true
```

Comment:
```suggestion
		// 字符串查找 有则返回true
```
Could you translate this english, easy for other maintainers for future refs